.travis.yml

```diff
@@ -2,7 +2,7 @@ language: python
 python:
   - "3.8"
 env:
-  - SECRET_KEY=somethinghereforciprtoexecuteintestbuilds
+  - SECRET_KEY=somethinghereforciprtoexecuteintestbuilds CORS_ORIGINS="*"
```

Comment:
Interesting, I didn't know yml allowed same line assignments. Could we break this out into two lines?
```suggestion
  - SECRET_KEY=somethinghereforciprtoexecuteintestbuilds
  - CORS_ORIGINS="*"
```

Reply:
I originally split the assignment into two lines, but Travis created two separate environments. Documentation states to include all of the environmental variables on a single line.

Reply:
Interesting. I did not know that.